charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -91,6 +91,27 @@ spec:
       {{- end }}
       volumeSnapshotLocation: []
     metrics:
+      prometheusRule:
+        additionalLabels: {{- .Values.monitoring.labels | toYaml | nindent 10 }}
+        enabled: true
+        spec:
+        - alert: VeleroBackupPartialFailures
+          annotations:
+            message: Velero backup {{ $labels.schedule }} has {{ $value | humanizePercentage }} partialy failed backups.
+          expr: velero_backup_partial_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+          for: 15m
+          labels:
+            severity: warning
+        - alert: VeleroBackupFailures
+          annotations:
+            message: Velero backup {{ $labels.schedule }} has {{ $value | humanizePercentage }} failed backups.
+          expr: velero_backup_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+          for: 15m
+          labels:
+            severity: warning
       serviceMonitor:
+      {{- if .Values.monitoring.prometheus.enabled }}
+        enabled: true
+      {{- end}}
```

Comment:
This is actually not needed, as this is autodetected by default

Reply:
Actually this is needed.

Resolved in call.

Reply:
Nvm, they are stupid